README.md

```diff
@@ -11,12 +11,17 @@ pushed to a configured git repository.
 
 ## Getting Started
 
-1. Install with `pip install airflow-windmill`
+1. python3 -V  # 3.6 and 3.7 should both be fine
```

Comment:
Should probably do generic instructions and then os/virtual env manager specific at some point

Reply:
Maybe. I'm not sure how I feel about dictating how someone manages their python libraries. What about making the default install instructions poetry install, since that will manage your venv anyway. Can have a link to poetry homepage and then a side note with pip install instructions as an alternative